releng/org.eclipse.january.releng.target/org.eclipse.january.releng.target.tpd

```diff
@@ -27,6 +27,7 @@ location "http://download.eclipse.org/tools/orbit/downloads/drops/S2016091616200
 	org.antlr.runtime [4.3.0,4.3.1) // CQ 11787
 	com.google.inject [3.0.0,3.0.1) // CQ 11788
 	org.apache.log4j [1.2.15,1.2.16) // CQ 11789
+	javax.measure.unit-api [1.0.0] // CQ 13959
```

Comment:
Its there now in an M build (is that better or worse than an S build?) Doesn't matter, we should have an R build in a few weeks with unit-api to upgrade to:

http://download.eclipse.org/tools/orbit/downloads/
specifically:
http://download.eclipse.org/tools/orbit/downloads/drops/M20170818183741/

Thanks to Erwin for getting this into Orbit.